wondrous-app/components/Common/NewInviteLinkModal/OrgInviteLink.tsx

```diff
@@ -0,0 +1,219 @@
+import { useEffect, useState } from 'react';
+import * as Sentry from '@sentry/nextjs';
+import {
+  StyledModal,
+  StyledBox,
+  TextHeading,
+  CloseButton,
+  PersonAddIconWrapper,
+  TextHeadingWrapper,
+  HeadingWrapper,
+  IconTextWrapper,
+  InviteButton,
+  InviteThruLinkLabel,
+  InviteThruLinkTextField,
+  InviteThruLinkButtonLabel,
+  InviteThruLinkInputWrapper,
+  StyledDivider,
+  InviteThruEmailLabel,
+  InviteThruEmailTextFieldButtonWrapper,
+  InviteThruEmailTextField,
+  InviteThruLinkSelect,
+  InviteThruLinkMenuItem,
+  InviteThruLinkFormControlSelect,
+  InviteThruEmailTextFieldSelectWrapper,
+  InviteThruEmailButtonLabel,
+  InviteThruLinkButtonSuccessLabel,
+  LinkSwitch,
+  TextSubheading,
+} from './styles';
+import PersonAddIcon from '../../Icons/personAdd';
+import { CopyIcon, CopySuccessIcon } from '../../Icons/copy';
+import { useMutation, useLazyQuery } from '@apollo/client';
+import { CREATE_ORG_INVITE_LINK } from 'graphql/mutations/org';
+import { GET_ORG_ROLES } from 'graphql/queries/org';
+import { useOrgBoard, usePodBoard } from 'utils/hooks';
+import { parseUserPermissionContext } from 'utils/helpers';
+import { LINK, PERMISSIONS } from 'utils/constants';
+
+export const putDefaultRoleOnTop = (roles, permissions) => {
```

Comment:
This function was copied from https://github.com/wondrous-dev/wondrous-frontend/blob/c5d48c0c1fc050bf0f3ec2cb4713433f6fafc13a/wondrous-app/components/Common/InviteLinkModal/OrgInviteLink.tsx#L39. 

Can you create import it from `components/Common/InviteLinkModal/OrgInviteLink.tsx` ?